db/migrate/20150330002335_create_organizations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOrganizations < ActiveRecord::Migration
+  def change
+    create_table :organizations do |t|
+      t.string :name, null: false, default: ''
+      t.string :slug, null: false, default: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.